### DISCUSSION TITLE
Hosts代理模式的443端口被虚拟机程序vmware-hostd占用

### DISCUSSION THREAD

**Discussion by SHUQIQAQ** [BeyondDimension/SteamTools#1156] [Ideas]
这个可以加一个功能吗？就是修改端口可以用443也可以用其他的端口，Hosts代理模式的443端口被虚拟机程序vmware-hostd占用，因为vmwore16.0之后共享虚拟机功能阉割了，不能修改端口了，而VMwore默认是443，所以可以加一个功能修改这个程序的端口就好了

**Comment by SHUQIQAQ** [comment]
Can I add a function to this? You can use 443 or other ports to modify the port. The 443 port of hosts proxy mode is occupied by the virtual machine program VMware host because vmwore16 After 0, the shared virtual machine function is castrated and the port cannot be modified, while vmword defaults to 443, so you can add a function to modify the port of this program

**Comment by rmbadmin** [comment]
hosts代理模式是必须使用443端口的，修改会导致无法使用，你还是需要优先考虑修改vm虚拟机共享的端口或者关闭虚拟机共享功能

**Reply by SHUQIQAQ** [reply]
可是16.0之后版本那个也不能修改了，功能被阉割了

**Reply by SHUQIQAQ** [reply]
我找找看有没有方法吧

**Reply by AigioL** [reply]
你可以参考 https://linzyjx.com/archives/65.html
